controllers/chat/ChatController.js

```diff
@@ -1,62 +1,168 @@
+      );
+
+      client.connectUser({ id: req.userId }, req.token);
+
+      if (!req.body.members.includes(req.userId))
+        req.body.members.push(req.userId);
+
+      const channel = client.channel('messaging', req.body.channelId, {
+        members,
+      });
+
+      await channel.create();
+      return res.status(200).json(responseSuccess('Success create channel'));
+    } catch (error) {
+      return res
+        .status(error.code ?? error.status ?? 400)
+        .json(responseError(error.message, error, error.code ?? error.status));
+    }
   },
 
   addChannelModerator: async (req, res) => {
     const schema = {
-      channelId: "string|empty:false",
-      members: "string[]|empty:false",
-    }
+      channelId: 'string|empty:false',
+      members: 'string[]|empty:false',
+    };
 
-    const validated = v.validate(req.body, schema)
-    if (validated.length) return res.status(403).json({
-      message: "Error validation",
-      error: validated
-    })
+    const validated = v.validate(req.body, schema);
+    if (validated.length)
+      return res.status(403).json({
+        message: 'Error validation',
+        error: validated,
+      });
 
-    let { channelId, members } = req.body
-    let { success, message } = await addModerators(channelId, members, req.token)
-    if (!success) return res.status(403).json({
-      success,
-      message: "Error creating channel",
-      error: message
-    })
+    let { channelId, members } = req.body;
+    let { success, message } = await addModerators(
+      channelId,
+      members,
+      req.token
+    );
+    if (!success)
+      return res.status(403).json({
+        success,
+        message: 'Error creating channel',
+        error: message,
+      });
 
     return res.status(200).json({
       success,
-      message: "Error creating channel",
-      error: message
-    })
+      message: 'Error creating channel',
+      error: message,
+    });
   },
 
   addMembers: async (req, res) => {
     let members = [];
     members.push(req.userId);
     let channel = await AddMembersChannel(
-      "messaging",
-      "morris-heights",
+      'messaging',
+      'morris-heights',
       members
     );
     return res
       .status(200)
-      .json(responseSuccess("Success add members channel", channel));
+      .json(responseSuccess('Success add members channel', channel));
+  },
+  sendAnonymous: async (req, res) => {
+    const schema = {
+      channelId: 'string|optional:true',
+      members: 'string[]|empty:false',
+      message: 'string|empty:false',
+    };
+    const validated = v.validate(req.body, schema);
+    if (validated.length)
+      return res.status(403).json({
+        message: 'Error validation',
+        error: validated,
+      });
+
+    const client = StreamChat.getInstance(
+      process.env.API_KEY,
+      process.env.SECRET
+    );
+
+    client.connectUser({ id: req.userId }, req.token);
+
+    if (!req.body.members.includes(req.userId))
```

Comment:
Mas @falhar this condition can be taken out, since members field is not needed anymore